informatiemodel/rdf/ontologie.ttl

```diff
@@ -146,11 +160,20 @@ zvg:BankgarantieOfWaarborgsom.uiterlijkeDatumGarantie a sh:PropertyShape;
 .
 zvg:BatenLastenCanons a <http://www.w3.org/2002/07/owl#Class>, sh:NodeShape;
     sh:name "BatenLastenCanons";
+    sh:property zvg:BatenLastenCanons.bedrag;
     sh:property zvg:BatenLastenCanons.overgenomenMetIngangVan;
     sh:targetClass zvg:BatenLastenCanons;
```

Comment:
targetclass is opzich overbodig omdat de subject uri zowel een nodeshape als een klasse is.
willen we de ontologie en de shapesgraph scheiden?